ostelco-ios-client/Coordinators/EKYC/DefaultEKYCCoordinator.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EKYCCoordinator.swift
+//  ostelco-ios-client
+//
+//  Created by Ellen Shapiro on 6/3/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import ostelco_core
+import PromiseKit
+import UIKit
+
+class DefaultEKYCCoordinator: EKYCCoordinator {
+    
+    static func coordinator(for country: Country,
```

Comment:
I tried to put this on the `EKYCCoordinator` protocol, but the compiler warned his has to be a static func on a concrete type or it can't actually generate a coordinator. Figured default was probably the better of the two to use as a starting point.